test/with_dummyserver/test_connectionpool.py

```diff
@@ -1055,5 +1055,13 @@ def test_pool_size_redirect(self):
         assert self.pool.num_connections == 1
 
 
+class TestBrokenPipeIgnore(HTTPDummyServerTestCase):
+    def setUp(self):
+        self.pool = HTTPConnectionPool(self.host, self.port)
+
+    def test_broken_pipe_ignore(self):
```

Comment:
Let's add this testcase under `TestHTTPConnectionPool` so it'll be re-used in other places automatically.

Reply:
And can we also add a testcase nearby for a chunked request as well?

Reply:
Cool ideas!